README.rst

```diff
@@ -54,8 +54,15 @@ Next to open a terminal at the project root and install the requirements for loc
     $ pip install -r requirements/local.txt
 
 Next to create MySQL database::
+    # if you are using Ubuntu 14.04, you may need to find a workaround for the following two commands
+    $ sudo systemctl start mariadb
+    $ sudo systemctl enable mariadb
     
-    $ echo "CREATE DATABASE poradnia CHARACTER SET utf8 COLLATE utf8_polish_ci;" | mysql
+    $ mysql_tzinfo_to_sql /usr/share/zoneinfo | sudo mysql mysql
```

Comment:
Z drugim mysql to faktycznie. Mój błąd. Przepraszam za kłopot.

Wysłano z urządzenia mobilnego.

12 lip 2016 5:20 PM "rwa" <notifications@github.com> napisał(a):

> In README.rst
> <https://github.com/watchdogpolska/poradnia/pull/259#discussion_r70460371>
> :
>
> >
> > -    $ echo "CREATE DATABASE poradnia CHARACTER SET utf8 COLLATE utf8_polish_ci;" | mysql
> > +    $ mysql_tzinfo_to_sql /usr/share/zoneinfo | sudo mysql mysql
>
> Zdaje się, że tutaj akurat musi zostać tak jak jest. Pierwsze mysql to
> komenda, drugie to baza (która akurat nazywa się mysql).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/watchdogpolska/poradnia/pull/259/files/7cd73a3ae100b23865c031e2e4acb303bfb57e0a#r70460371>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADc2r3zL7dYRinW1xHXl3oMxLYTkivImks5qU7CPgaJpZM4JKT-X>
> .
>